index.js

```diff
@@ -6,17 +6,19 @@ var loaderUtils = require("loader-utils");
 var mime = require("mime");
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
-	var query = loaderUtils.getOptions(this) || {};
-	var limit = (this.options && this.options.url && this.options.url.dataUrlLimit) || 0;
-	if(query.limit) {
-		limit = parseInt(query.limit, 10);
+	var options =  Object.assign({}, loaderUtils.getOptions(this));
```

Comment:
`loaderUtils.getOptions(this) || {};` 

`Object.assign` is slower and there is nothing (e.g `defaults`) to  assign